R/use_remake.R

```diff
@@ -0,0 +1,24 @@
+##' Add a remake file template
+##' 
+##' This function adds a simple remake file template to a 
+##' target directory.
+##' 
+##' @param dir directory in which to create. Defaults to
+##'   current directory.
+##' @return This function is primarily useful for its side 
+##'   effect, which is to add a template remake.yml file to 
+##'   the home directory.
+##' @export
+use_remake <- function(dir = ".") {
+  remake_filename <- "remake"
+  path <- file.path(dir, paste0(remake_filename, ".yml"))
```

Comment:
Any particular reason for this construct, given that `remake_filename` is a constant?